user_manual/groupware/mail.rst

```diff
@@ -156,3 +156,16 @@ Unsubscribe from a mailing list
 .. versionadded:: 3.1
 
 Some mailing lists and newsletters allow to be unsubscribed easily. If the Mail app detects messages from such a sender, it will show an *Unsubscribe* button next to the sender information. Click and confirm to unsubscribe from the list.
+
+Snooze
+~~~~~~
+
+.. versionadded:: 3.4
+
+Snoozing a message or thread moves it in a dedicated mailbox until the selected snooze date is reached and the message or thread is moved back to the original mailbox.
+
+1. Open action menu of an envelope or thread
+2. Click *Snooze*
+3. Select how long the message or thread should be snoozed
+
+.. note:: If AJAX is selected for cron job execution in the admin settings, the snooze feature is deactivated.
```

Comment:
not sure if we want to give this information to users. we could move this information into the mail admin docs

Reply:
:+1: Are there any other features that are disabled with ajax (beside snooze and scheduled sending) worth mentioning?

Reply:
those should be the only two